script/Deploy.s.sol

```diff
@@ -75,9 +75,12 @@ contract DeployMainnet is MainnetParams, Deploy {
 
   function setupEnvironment() public virtual override updateParams {
     // Setup oracle feeds
-    IBaseOracle _ethUSDPriceFeed = chainlinkRelayerFactory.deployChainlinkRelayer(OP_CHAINLINK_ETH_USD_FEED, 1 hours);
-    IBaseOracle _wstethETHPriceFeed =
-      chainlinkRelayerFactory.deployChainlinkRelayer(OP_CHAINLINK_WSTETH_ETH_FEED, 1 hours);
+    IBaseOracle _ethUSDPriceFeed = chainlinkRelayerFactory.deployChainlinkRelayer(
+      OP_CHAINLINK_ETH_USD_FEED, OP_CHAINLINK_SEQUENCER_UPTIME_FEED, 1 hours
```

Comment:
> hmmmm is it one for all chainlink or one for each feed?

It is one per network.

> i wonder if we should include it on the factory, but since we're handling all immutable oracles it makes sense to deploy it like this
> if they change their uptime feed, we need to redeploy the oracle (and not the factory)

I agree.